components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -56,17 +58,22 @@ import mozilla.components.concept.engine.permission.SitePermissions.Status.ALLOW
 import mozilla.components.concept.engine.permission.SitePermissions.Status.BLOCKED
 import mozilla.components.concept.engine.permission.SitePermissionsStorage
 import mozilla.components.feature.sitepermissions.SitePermissionsFeature.DialogConfig
+import mozilla.components.feature.tabs.TabsUseCases.SelectOrAddUseCase
 import mozilla.components.lib.state.ext.flowScoped
 import mozilla.components.support.base.feature.LifecycleAwareFeature
 import mozilla.components.support.base.feature.OnNeedToRequestPermissions
 import mozilla.components.support.base.feature.PermissionsFeature
 import mozilla.components.support.ktx.android.content.isPermissionGranted
 import mozilla.components.support.ktx.kotlin.getOrigin
+import mozilla.components.support.ktx.kotlin.stripDefaultPort
 import mozilla.components.support.ktx.kotlinx.coroutines.flow.filterChanged
 import mozilla.components.support.ktx.kotlinx.coroutines.flow.ifChanged
 import java.security.InvalidParameterException
 
 internal const val FRAGMENT_TAG = "mozac_feature_sitepermissions_prompt_dialog"
+@VisibleForTesting
+internal const val STORAGE_ACCESS_DOCUMENTATION_URL =
+    "https://developer.mozilla.org/en-US/docs/Web/API/Storage_Access_API"
```

Comment:
nit: Maybe could allow apps to pass their own link and make this one as the default value, as apps may have their own SUMO article page.

Reply:
Since the entire functionality is based on the Mozilla API and the Gecko / GVs implementation details I think it would be okay to hardcode the url. 
Also this AC feature seems like a standalone one with a general API so putting this highly specific detail in the constructor for just one permission would make it a bit more cumbersome?

Reply:
Good point👌🏽 . Let's leave it as it's :)